forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -111,6 +122,14 @@ pub enum SnapshotCommands {
         #[structopt(long)]
         force: bool,
     },
+    /// Validates the snapshot.
+    Validate {
+        /// Number of block headers to validate from the tip
+        #[structopt(long)]
+        validate_height: i64,
```

Comment:
How about we name this `recent_stateroots` to match the argument in the export command? It should also default to 2k.

Reply:
added